packages/editor/src/components/block-list/style.scss

```diff
@@ -780,11 +780,18 @@
 	> .editor-block-list__insertion-point {
 		position: absolute;
 		top: -$block-padding - $block-spacing / 2;
-		// Matches the whole empty space between two blocks
+
+		// Matches the whole empty space between two blocks.
 		height: $block-padding * 2;
 		bottom: auto;
-		left: -$border-width;
-		right: -$border-width;
+		left: 0;
+		right: 0;
+
+		// Stack with block borders beyond mobile.
```

Comment:
I'm not quite sure what this comment means, and it references code aimed at a mobile viewport but talks about what happens *outside* the mobile viewport. I get that it means mobile behaves one way and others differently, but I think this comment could be clarified 😄

Reply:
Solid point, will rearrange and clarify that a bit. 

THANKs for all the fixes and review!

Reply:
Addressed your comment in https://github.com/WordPress/gutenberg/pull/9010/commits/f5c3696b191263bc9309f28b215d4e801e5cde63